en/core-libraries/inflector.rst

```diff
@@ -13,6 +13,57 @@ statically. Example:
 You can try out the inflections online at `inflector.cakephp.org
 <http://inflector.cakephp.org/>`_.
 
+Summary of Inflector Methods and Their Output
+===================================================
+
+Quick summary of the Inflector built-in methods and the results they output
```

Comment:
Sounds a bit redundant, may be:
``When provided a multi-word argument, the Inflector methods output the following:``